src/client/sandbox/child-window/index.ts

```diff
@@ -28,9 +28,10 @@ export default class ChildWindowSandbox extends SandboxBase {
     }
 
     private static _shouldOpenInNewWindowOnElementAction (el: HTMLLinkElement | HTMLAreaElement | HTMLFormElement, defaultTarget: string): boolean {
-        const target = this._calculateTargetForElement(el);
+        const target               = this._calculateTargetForElement(el);
+        const hasDownloadAttribute = nativeMethods.getAttribute.call(el, 'download');
```

Comment:
I guess we can use fast forward here.
```js
private static _shouldOpenInNewWindowOnElementAction (....) {
     const hasDownloadAttribute = nativeMethods.getAttribute.call(el, 'download');

     if (hasDownloadAttribute)
         return false;

  ...
```

Reply:
Well, me personally, I do not see any advantages or disadvantages of any of these two approaches

Reply:
Advantage is code readability.

Compare:
```js
private static _shouldOpenInNewWindowOnElementAction (): boolean {
    const target               = this._calculateTargetForElement(el);
    const hasDownloadAttribute = nativeMethods.getAttribute.call(el, 'download');

    return hasDownloadAttribute ? false : this._shouldOpenInNewWindow(target, defaultTarget);
}


private static _shouldOpenInNewWindowOnElementAction (): boolean {
     const hasDownloadAttribute = nativeMethods.getAttribute.call(el, 'download');

    if (hasDownloadAttribute)
        return false;

    const target = this._calculateTargetForElement(el);

    return this._shouldOpenInNewWindow(target, defaultTarget);
}


```